ws-nextjs-app/pages/[service]/watch/[id]/LiveTvPageLayout.tsx

```diff
@@ -1,25 +1,77 @@
-import React from 'react';
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
+import React, { use } from 'react';
+import MediaLoader from '#app/components/MediaLoader';
+import { Curation } from '#app/models/types/curationData';
+import LiveTVCuration from '#app/components/Curation';
+import { data as liveTvFixture } from '#data/dari/watch/bbc_afghan_tv/live.json';
+import Heading from '#app/components/Heading';
+import Text from '#app/components/Text';
+import MetadataContainer from '#app/components/Metadata';
+import { ServiceContext } from '#app/contexts/ServiceContext';
 import { LiveTVPageProps } from './types';
+import styles from './styles';
 // import ChartbeatAnalytics from '#app/components/ChartbeatAnalytics';
 // import ATIAnalytics from '#app/components/ATIAnalytics';
-// import MetadataContainer from '#app/components/Metadata';
 
-export default function LiveTvLayout({ pageType, service }: LiveTVPageProps) {
+export default function LiveTvLayout({ pageData }: LiveTVPageProps) {
+  const { lang } = use(ServiceContext);
+  if (!pageData) {
+    // @ts-expect-error liveTvFixture used for development purposes only
+    // eslint-disable-next-line no-param-reassign
+    pageData = liveTvFixture;
+  }
+  const { curations, mediaBlock } = pageData;
```

Comment:
can we pull out title and description here as well? as they're used more than once each below